src/data/reducers/connectionStatus.js

```diff
@@ -0,0 +1,10 @@
+import { PING_RESPONSE } from '../actions/pingStudio';
+
+export default function connectionStatus(state = null, action) {
```

Comment:
This function is a reducer (http://redux.js.org/docs/basics/Reducers.html), the final step in the data flow process. Anything it returns gets stored as application state. Its name `connectionStatus` becomes the root-level key in the Redux store, so anything this function returns is stored under `connectionStatus`.

Reply:
While watching the egghead tutorial, I saw Dan Abramov using the ES6 arrow function notation for his reducer functions. Is there an advantage to doing it that way? Would it be better to use that notation, or is there a reason to keep it this way? Just curious!